src/commands/remind.ts

```diff
@@ -0,0 +1,124 @@
+import Client from "../struct/Client"
+import Message from "../struct/discord/Message"
+import Args from "../struct/Args"
+import Command from "../struct/Command"
+import Roles from "../util/roles"
+import Reminder from "../entities/Reminder"
+import formatTimestamp from "../util/formatTimestamp"
+
+export default new Command({
+    name: "remind",
+    aliases: [],
+    description: "List and manage reminders.",
+    permission: [Roles.MANAGER],
+    usage: "",
+    subcommands: [
+        {
+            name: "add",
+            description: "Add a reminder.",
+            permission: [Roles.MANAGER],
+            usage: "<channel> <interval> <message>"
+        },
+        {
+            name: "edit",
+            description: "Edit a reminder.",
+            permission: [Roles.MANAGER],
+            usage: "<id> <message>"
+        },
+        {
+            name: "delete",
+            description: "Delete a reminder.",
+            permission: [Roles.MANAGER],
+            usage: "<id>"
+        }
+    ],
+    async run(this: Command, client: Client, message: Message, args: Args) {
+        const subcommand = args.consume().toLowerCase()
+
+        if (subcommand === "list" || !subcommand) {
+            const reminders = await Reminder.find()
+            const tidy: Record<string, { channel: string; message: string; end: Date }> =
+                {}
+
+            for (const reminder of reminders) {
+                if (!tidy[reminder.id])
+                    tidy[reminder.id] = { channel: "", message: "", end: new Date() }
+
+                tidy[reminder.id].channel = reminder.channel
+                tidy[reminder.id].message = reminder.message
+                tidy[reminder.id].end = reminder.end
+            }
+
+            let list = ""
+            for (const [id, { channel, message, end }] of Object.entries(tidy)) {
+                list += `${id}: \u200B \u200B <#${channel}> (next reminder ${formatTimestamp(end, "R")}) - ${message}\n`
```

Comment:
change the ID format to be more consistent with the other ordered list commands, remove "reminder" to make it shorter, and use an em-dash! those are cool
```suggestion
                list += `**#$id:** <#${channel}> (next ${formatTimestamp(end, "R")}) — ${message}\n`
```